tutorials/peft-curation-with-sdg/README.md

```diff
@@ -44,9 +44,18 @@ python tutorials/peft-curation-with-sdg/main.py \
 # To control the amount of synthetic data to generate
 python tutorials/peft-curation-with-sdg/main.py \
     --api-key YOUR_BUILD.NVIDIA.COM_API_KEY \
-    --device gpu \
-    --synth-gen-rounds 2 \ # Do 2 rounds of synthetic data generation
-    --synth-gen-ratio 0.5  # Generate synthetic data using 50% of the real data
+    --device gpu \  # Use the GPU and enable semantic deduplication
+    --synth-gen-rounds 1 \ # Do 1 round of synthetic data generation
+    --synth-gen-ratio 0.001 \  # Generate synthetic data using 0.1% of the real data
+    --synth-gen-model "meta/llama-3.1-405b-instruct" # Use LLaMa 3.1 405B
+
+# To control the amount of synthetic data to generate
```

Comment:
Nit: Could you modify this comment and the comment above to reference the model that the command is using? Something like
```python
# To control the amount of synthetic data to generate using Llama 3.1 405B
```
and
```python
# To control the amount of synthetic data to generate using Nemotron-4 340B
```

Reply:
Done.